pensando_psm/info.json

```
@@ -0,0 +1,544 @@
+{
+    "name": "pensando_psm",
+    "version": "1.0.0",
+    "label": "Pensando Policy Service Manager",
+    "description": "Perform actions on Pensando Policy Service Manager",
+    "publisher": "Fortinet",
+    "icon_small_name": "pensando_logo_small.jpg",
+    "icon_large_name": "pensando_logo_large.jpg",
+    "category": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "Server Address",
+                "type": "text",
+                "name": "server_address",
+                "required": true,
+                "visible": true,
+                "editable": true
```

```suggestion
                "editable": true,
                "description": "Server Address of the Pensando PSM server to which you will connect and perform the automated operations."
```
Please add description to all parameters. This will help us to generate doc.